src/pages/RepoPage/RepoPage.tsx

```diff
@@ -224,18 +229,20 @@ function RepoPage() {
     },
   })
 
-  const coverageEnabled = repoOverview?.coverageEnabled
-  const bundleAnalysisEnabled = repoOverview?.bundleAnalysisEnabled
+  const coverageEnabled =
+    repoOverview?.coverageEnabled ?? repoData?.isCoverageEnabled
+  const bundleAnalysisEnabled =
+    repoOverview?.bundleAnalysisEnabled ?? repoData?.isBundleAnalysisEnabled
   const jsOrTsPresent = repoOverview?.jsOrTsPresent
   const isCurrentUserActivated = repoData?.isCurrentUserActivated
   const isRepoActive = repoData?.repository?.active
   const isRepoActivated = repoData?.repository?.activated
-  const isRepoPrivate = !!repoData?.repository?.private
-
+  const isRepoPrivate =
+    !!repoData?.repository?.private ?? repoData?.isRepoPrivate
   if (!refetchEnabled && !isRepoActivated) {
     setRefetchEnabled(true)
   }
-  if (!repoData?.repository) return <NotFound />
+  if (!repoData) return <NotFound />
```

Comment:
why this change?

Reply:
We could get a null repository response if the owner is not activated but we don't want to show a `NotFound` in that case.